tests/security/usbguard/usbguard.pm

```diff
@@ -0,0 +1,137 @@
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test 'usbguard' can work:
+#          '# systemctl enable usbguard.service',
+#          '# systemctl start usbguard.service',
+#          '# systemctl status/is-active usbguard.service',
+#          '# usbguard generate-policy', '# usbguard allow-device',
+#          '# usbguard list-devices', '# usbguard block-device'
+#          '# check/cat '/var/log/usbguard/usbguard-audit.log',
+#          '# usbguard add-user', '# usbguard remove-user',
+#          '# usbguard get-parameter', '# usbguard set-parameter'
+#          '# usbguard generate/list/install/remove rules'
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#102566, tc#1769830
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my $out = '';
+    my $user = 'usbguard';
+    my $allow = 'allow';
+    my $block = 'block';
+    my $msg = 'Tablet';
+    my $f_rules_default = '/etc/usbguard/rules.conf';
+    my $f_rules_test = '/etc/usbguard/rules_test.conf';
+    my $f_IPCAccessControl = '/etc/usbguard/IPCAccessControl.d/\:' . "$user";
+    my $f_usbguard_audit_log = '/var/log/usbguard/usbguard-audit.log';
+
+    select_console 'root-console';
+
+    # 0. Set up environment
+    # Install usbguard packages
+    zypper_call('in libusbguard1 usbguard usbguard-devel usbguard-tools', timeout => 900);
+
+    # Start audit service
+    systemctl('restart auditd.service');
+
+    # Create /etc/nsswitch.conf for Tumbleweed otherwise there will be an Error like
+    # "usbguard-daemon: NSSwitch parsing: /etc/nsswitch.conf: No such file or directory"
+    # when start usbguard service
+    assert_script_run('touch /etc/nsswitch.conf');
```

Comment:
cp /usr/etc/nsswitch.conf /etc/nsswitch.conf

Reply:
Thanks for the info.
Since this file doesn't block any test, I would like to chose "touch" (it is harmless and simple)

Reply:
I would suggest you copy the file rather than create a blank one

Reply:
Thanks for the info
I prefer "touch" as it is really simple (especially make coding simple) and harmless.